.ci/integrationTest.postgres.sh

```diff
@@ -5,11 +5,17 @@ dockerCompose() {
     docker compose -f ./.ci/docker-compose.test.yml -f ./.ci/docker-compose.test.postgres.yml "$@"
 }
 
-dockerCompose down
-dockerCompose build
-dockerCompose up -d
-dotnet restore "Backbone.sln"
-dotnet build --no-restore "Backbone.sln"
+{
+  dockerCompose down;
+  dockerCompose up -d --no-build
+} &
+{
+  dotnet restore "Backbone.sln";
+  dotnet build --no-restore "Backbone.sln"
+}
+wait
+
 export CONSUMER_API_BASE_ADDRESS="http://localhost:5000"
 export ADMIN_API_BASE_ADDRESS="http://localhost:5173"
-dotnet test --no-restore --no-build --filter "Category=Integration&TestCategory!~ignore" --logger "GitHubActions;summary.includeNotFoundTests=false" "Backbone.sln"
+
+dotnet test --no-restore --no-build --filter "Category=Integration&TestCategory!~ignore" --logger "GitHubActions;summary.includePassedTests=true;summary.includeNotFoundTests=false" "Backbone.sln"
```

Comment:
Why did you add `summary.includePassedTests=true`?

Reply:
I'm not sure what was copied in and what was merged in